doc/source/whatsnew/v0.17.0.txt

```diff
@@ -110,6 +110,7 @@ Other enhancements
 
 - ``pd.merge`` will now allow duplicate column names if they are not merged upon (:issue:`10639`).
 
+- ``msgpack`` submodule updated to 0.4.6 (:issue:`10581`)
 .. _whatsnew_0170.api:
```

Comment:
ok, seems fine then. Do we have any tests to unpacking with ``pandas.msgpack.unpackb`` after packing with ``pandas.io.packers.pack``? (e.g. trying to unpack the pandas objects themsevles?). I think you would get a weird structure, but it should work.